rapids/README.md

```diff
@@ -0,0 +1,96 @@
+
+RAPIDS is a relatively young project with APIs evolving quickly. If you encounter unexpected errors or have feature requests, please file them at the relevant [RAPIDS repo](https://github.com/rapidsai).
+
+### Options
+
+By default, the master node also runs dask-cuda-workers. This is useful for smaller scale jobs- processes run on 4 GPUs in a single node will usually be more performant than when run on the same number of GPUs in separate server nodes (due to higher communication costs).
+
+If you want to save the master's GPU(s) for other purposes, this behavior is configurable via a metadata key using `--metadata`.
+
+* `master-worker=false` - whether to run dask-cuda-workers on the master node
+
+For example:
+```bash
+DATAPROC_BUCKET=dataproc-initialization-actions
+
+gcloud beta dataproc clusters create <CLUSTER_NAME> \
+--zone us-east1-c \
+--master-accelerator type=nvidia-tesla-t4,count=4 \
+--master-machine-type n1-standard-32 \
+--worker-accelerator type=nvidia-tesla-t4,count=4 \
+--worker-machine-type n1-standard-32 \
+--metadata "master-worker=false" \
+--bucket $DATAPROC_BUCKET \
+--initialization-actions gs://$DATAPROC_BUCKET/rapids/install-gpu-driver.sh,gs://$DATAPROC_BUCKET/rapids/install-rapids.sh,gs://$DATAPROC_BUCKET/rapids/launch-dask.sh
+```
+
+By default, these initialization actions assume you are using a Tesla T4 GPU. If you are using other GPU types or want to run a different driver version, [find the appropriate driver download URL](https://www.nvidia.com/Download/index.aspx?lang=en-us) for your driver's `.run` file.
+
+* `gpu-driver-url=http://us.download.nvidia.com/tesla/410.104/NVIDIA-Linux-x86_64-410.104.run` - specify an alternate driver download URL
+
+For example:
+
+```bash
+DATAPROC_BUCKET=dataproc-initialization-actions
+
+gcloud beta dataproc clusters create <CLUSTER_NAME> \
+--zone us-east1-c \
+--master-accelerator type=nvidia-tesla-t4,count=4 \
+--master-machine-type n1-standard-32 \
+--worker-accelerator type=nvidia-tesla-t4,count=4 \
+--worker-machine-type n1-standard-32 \
+--metadata "gpu-driver-url=http://us.download.nvidia.com/tesla/410.104/NVIDIA-Linux-x86_64-410.104.run" \
+--initialization-actions gs://$DATAPROC_BUCKET/rapids/install-gpu-driver.sh,gs://$DATAPROC_BUCKET/rapids/install-rapids.sh,gs://$DATAPROC_BUCKET/rapids/launch-dask.sh
+```
+
+## Important notes
+* RAPIDS is only supported on Pascal or newer GPU architectures.
+* You must set a GPU accelerator type for both master and worker nodes, else the GPU driver install will fail and the cluster will report an error state
+* We recommend an n1-standard-32 worker machine type or better to ensure sufficient host-memory for buffering data to and from GPUs.
```

Comment:
During testing I noticed that if you use only 1 accelerator card, you get following error message if n1-standard-32 machine type is used.
`Invalid value for field 'resource.guestAccelerators[0].acceleratorCount': '1'. At most 24 vCPUs can be used along with 1 accelerator cards in an instance..`

Reply:
Added comments on this behavior in "Important notes" section